### PR TITLE
Create staff members

### PR DESCRIPTION
**Pull request by Rinbo**
PT: https://www.pivotaltracker.com/story/show/161759765

- Adds staff members to staff section through a form on the schedule page
- New staff members gets saved in the database and rendered on screen instantly.

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     fill_in(field, with: text)
 end
 
-Given("I fill in {string} with 1st of Jan, 2018") do |field|
+Given("I fill in {string} with Jan, 2018") do |field|
     element = page.find_by_id(field)
     element.send_keys :up
     element.send_keys :tab
```
Comment (danmalmx) [inline features/step_definitions/basic_steps.rb:16]: What exactly are we checking for here? If up and tab button works to change date?

File: features/schedules.feature [hunk 2]
```diff
@@ -15,7 +15,7 @@ Feature: Schedule model
   @javascript
   Scenario: User visits schedule page and creates a schedule
     When I click "New Schedule"
-    And I fill in "date-input" with 1st of Jan, 2018
+    And I fill in "date-input" with Jan, 2018
```
Comment (danmalmx) [inline features/schedules.feature:18]: Why did we take away checking for a specific date?

File: features/step_definitions/basic_steps.rb (hunk 1 above)
Comment (Rinbo) [reply features/step_definitions/basic_steps.rb:16]: Filling in the form with the Month and Year. Simulating the key strokes to make that happen as in fill_in form didn't work for a dual field.

File: features/schedules.feature (hunk 2 above)
Comment (Rinbo) [reply features/schedules.feature:18]: Because a planning period is for a given month. Not a specific date.

**Pull request merged**